crates/sui-network/tests/bootstrap.rs

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
```

Comment:
We're leveraging some code generation based off of some `.proto` files. The general approach to doing code generation is to run it via a `build.rs` file and then do an `include!` macro to bring the actual source in. One issue I found with that is that ide support breaks and you can no longer easily see the generated code. Instead this "integration" tests assumes the role of the `build.rs` file doing the codegen and sticking the output in the `src/generated` directory so that you can easily see the generated code as well as IDE support for things like go-to-defintion work